packages/zoe/src/contractFacet/vatRoot.js

```diff
@@ -35,6 +35,7 @@ export function buildRootObject(powers, _params, testJigSetter = undefined) {
     zoeInstanceAdmin,
     instanceRecordFromZoe,
     issuerStorageFromZoe,
+    privateArgs,
```

Comment:
I like to be notationally explicit about option params, in the code as well as the type.

```suggestion
    privateArgs = undefined,
```

This also occurs several other places below

Reply:
Done!